index.html

```diff
@@ -121,12 +120,10 @@ <h2>
         Badge display
       </h2>
       <p>
-        When the application is <a data-cite=
-        "appmanifest#dfn-installing">installed</a> and the application's flag
-        is <a>set</a>, the user agent SHOULD display the application's badge
-        alongside the symbolic representation of the application in the user's
-        operating system (for example, as a small overlay on top of the
-        application's icon).
+        When the application's flag is <a>set</a>, the user agent SHOULD display
```

Comment:
I think this fixes #12

Reply:
Hmm I think #12 was about completely removing the notion of this being related to an installed app. e.g. if it worked in a tab, it could potentially be used without the app being installed

Reply:
Hmm, okay. I don't know how I feel about that. The one Badge per application example gets kind of hazy.

Reply:
How would we treat this in per tab land? I think there has to be some kind of guideline about what gets badged, and I wouldn't like it to work differently based on whether or not the app is installed.

Reply:
That's true. Application scope is the simplest thing to use. Let's keep it and comment on the bug.

Reply:
This has come up again in #35